### DISCUSSION TITLE
BUG: check for notifications update on every "window focus" event

### DISCUSSION THREAD

**Discussion by ashuvssut** [community/community#126776] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

<img width="643" alt="image" src="https://github.com/community/community/assets/60546840/09d30712-6688-4157-86c8-b0d395cc4aa3">

This blue dot is a false notification in the screenshot. I have already read all of my notifications and then I went back to my original page and I still see that blue dot. The blue dot will only disappear if i refresh my page.

An easy fix would be to fetch notification update on every window focus event (fetch notifications when browser tab is switched)

**Comment by ebndev** [comment]
Hi @ashuvssut, thanks for flagging this!

I am keeping a log of this issue as other users have mentioned this as well. The GitHub product team will evaluate the feedback but will not be able to respond to every submission. You can keep an eye on our [public roadmap](https://github.com/github/roadmap/projects/1) to see the kinds of work we are focusing on for the coming year. Smaller fixes won’t be listed, but are often included when work is happening adjacent to it.

In the meantime, please see [this previous discussion](https://github.com/orgs/community/discussions/123129#discussioncomment-9451090) for possible solutions.

Feel free to ask questions on the original threads if you run into an issue.

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.
